noir-projects/aztec-nr/aztec/src/keys/getters.nr

```diff
@@ -90,16 +89,3 @@ fn fetch_and_constrain_keys(address: AztecAddress) -> PublicKeys {
 
     public_keys
 }
-
-// We get the full struct Nullifier Keys here
-pub fn get_nullifier_keys(npk_m_hash: Field) -> NullifierKeys {
-    let nullifier_keys = get_nullifier_keys_oracle(npk_m_hash);
-    assert_eq(nullifier_keys.master_nullifier_public_key.hash(), npk_m_hash);
```

Comment:
This was just semi-constrained as the nsk_app was not checked. I initially though this is vulnerability but it turned out to not be case because we used this in contexts where it was not necessary (`Note::compute_nullifier_without_context`).